contracts/bonding/BondingManager.sol

```diff
@@ -152,6 +155,8 @@ contract BondingManager is IBondingManager, Manager {
         t.pendingFeeShare = _feeShare;
         t.pendingPricePerSegment = _pricePerSegment;
 
+        registeredTranscoders.push(msg.sender);
```

Comment:
Probably should add a check so that a transcoder doesn't get added if it already exists.